Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -139,6 +139,7 @@ All the other settings from the gateway section will be ignored, except for `sec
 The [zone](../../Storage-Configuration/Object-Storage-RGW/ceph-object-multisite.md) settings allow the object store to join custom created [ceph-object-zone](ceph-object-multisite-crd.md).
 
 * `name`: the name of the ceph-object-zone the object store will be in.
+* `endpoint`: the endpoint of the object-store to be added to the zone's endpoint list.
```

Comment:
More description is needed about what the "zone's endpoint list" would be. When does the user need to set this? What if it's not set? Or is it required?

Reply:
Agreed. @alimaredia could you help provide the best description here? To my understanding, this should be set to the publicly-available endpoint of the CephObjectStore associated with the Zone if the user doesn't wish to use the ClusterIP Service created by Rook. For example, if a user creates a LoadBalancer Service for ingress to the CephObjectStore's RGWs, this should be set to the IP or DNS name of the LoadBalancer. This value should match the value which is used by clients, including other Ceph clusters which have RGW multisite configured to pull from this zone.

It's a lot, but I think that information is sufficient to clarify things for any user.

Reply:
We use istio-ingress with a sub domain and virtual service for ceph multi-site.

So, something like 'objectstore.mycluster.example.com'.  The virtual service points at the rook-ceph k8s service (rook-ceph-rgw-rook-ceph).

So, the endpoint we set (via toolox as the operator doesn't currently support) is 'objectstore.mycluster.example.com'

What's important to note, is that the rgw is reachable via a variety of means, but the endpoint needs to be setup to support the replication traffic.  So, if you have 2 separate k8s clusters, user whatever respective name or IP is reachable from the remote cluster.  In our case, that's something like 'objectstore.eastcluster.example.com' and 'objecstore.westcluster.example.com'.  RGW will still be reachable internal to the k8s cluster via the normal svc address or any other domain that points to the svc.

Reply:
Both of the descriptions/notes @BlaineEXE and @mgugino-uipath have are fantastic and I don't have much to add more of a description to add. @BlaineEXE would you like for me to write something up and add it to this file in this PR?

Reply:
@alimaredia Yes please